packages/access-client/src/agent-use-cases.js

```diff
@@ -210,7 +208,7 @@ export async function waitForAuthorizationOnSocket(access, opts = {}) {
  * `access/claim` capability and waiting for the result to include
  * a session delegation.
  *
- * @type AuthorizationWaiter
+ * @type import('./types.js').AuthorizationWaiter<{interval?: number}>
```

Comment:
previously `AuthorizationWaiter` was only a function that would wait a single time, with no notion of an interval. e.g. when waiting using a socket, there is no interval.

Previously, you could always do `(interval: number) => AuthorizationWaiter`